src/Functions.h

```diff
@@ -181,13 +181,15 @@ namespace Functions {
     }
 
     std::string GetAmxPublicName(AMX *amx, int index) {
-        char name[32]{};
+        int length;
+        amx_NameLength(_amx, &length);
```

Comment:
> '_amx' was not declared in this scope

Reply:
sorry for the oversight, thanks for fixing it yourself